wallet/src/main/scala/org/bitcoins/wallet/models/DLCFundingInputDb.scala

```diff
@@ -13,6 +14,8 @@ case class DLCFundingInputDb(
     isInitiator: Boolean,
     outPoint: TransactionOutPoint,
     output: TransactionOutput,
+    redeemScriptOpt: Option[ScriptPubKey],
```

Comment:
scaladoc explaining what these new ones are would be great as it isn't entirely obvious as they are conveyed differently from the others ... I think?